c/polyjuice.h

```diff
@@ -87,6 +87,7 @@ int gw_increase_nonce(gw_context_t *ctx, uint32_t account_id, uint32_t *new_nonc
 int handle_message(gw_context_t* ctx,
                    uint32_t parent_from_id,
                    uint32_t parent_to_id,
+                   uint128_t gas_price,
```

Comment:
Please define this variable in `polyjuice_globals.h` instead of passing around, since `gas_price` is a transaction level arguments.